.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: "\U0001F41B Bug report"
+about: Create a report to help us improve
+
+---
+
+<!--
+Thank you for reporting a possible bug in Node.js.
+
+Please fill in as much of the template below as you're able.
+
+Version: output of `node -v`
+Platform: output of `uname -a` (UNIX), or version and 32 or 64-bit (Windows)
+Subsystem: if known, please specify affected core module name
+
+If possible, please provide code that demonstrates the problem, keeping it as
```

Comment:
nit: `provide code` -> `provide the code`

Reply:
I'm mildly opposed to this change too. Just `provide code`.